t/lib/t/MusicBrainz/DataStore/RedisMulti.pm

```diff
@@ -23,10 +23,10 @@ stores. Consistency among all the stores is checked after each operation.
 
 # Initialize tests
 my $args1 = DBDefs->DATASTORE_REDIS_ARGS;
-$args1->{database} = $args1->{test_database};
+$args1->{database} = DBDefs->REDIS_TEST_DATABASE;
 
 my $args2 = DBDefs->DATASTORE_REDIS_ARGS;
-$args2->{database} = $args2->{test_database} + 1;
+$args2->{database} = DBDefs->REDIS_TEST_DATABASE + 1;
```

Comment:
Given this here, should the docs in DBDefs say that both `REDIS_TEST_DATABASE` and `REDIS_TEST_DATABASE + 1` will be completely erased on each test run? Or should we define `REDIS_SECOND_TEST_DATABASE`? You know someone will use 0 for test and 1 for production some day.

Reply:
Yeah, I think we should just mention that any Redis database numbers greater than or equal to `REDIS_TEST_DATABASE` may be used for tests.